source/submit-feedback.txt

```diff
@@ -6,10 +6,79 @@ Submit Feedback
 
 .. default-domain:: mongodb
 
+.. facet:: 
+   :name: genre
+   :values: tutorial 
+
+.. meta:: 
+   :description: Submit feedback on the MongoDB for IntelliJ Plugin JIRA project.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44968.
+To report a bugs, request a feature, or submit general feedback, follow the 
+procedures on this page to open a ticket in the 
+`MongoDB for IntelliJ Plugin JIRA project <https://jira.mongodb.org/projects/INTELLIJ/issues/>`__.
+
+Before You Begin 
+----------------
+
+To submit feedback and vote on issues, you must be logged in to a JIRA account.
+
+Steps 
+-----
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Open the ticket creation dialog.
+
+      To open a new ticket, click :guilabel:`Create` in the JIRA top navigation 
+      bar.
+
+   .. step:: Specify the issue type.
+
+      Open the :guilabel:`Issue Type` drop-down menu and select the appropriate 
+      issue type: 
+
+      - If you encounter a bug, select :guilabel:`Bug`.
+
+      - If you have a new feature request, select :guilabel:`New Feature`.
+
+   .. step:: Provide a summary. 
+
+      In the :guilabel:`Summary` field, provide a brief summary of the bug or
+      feature request. 
+      
+   .. step:: Provide a description. 
+      
+      In the :guilabel:`Description` field, you can provide more detailed 
+      information, including the exact steps needed to reproduce the issue. 
+
+   .. step:: Set the label field.
+
+      In the :guilabel:`Labels` field, include the ``internal-preview`` label.
+
+   .. step:: (*Optional*) Add logs in the Attachment field.
+
+      Logs can help navigate edge cases and diagnose issues in the 
+      {+intellij-short+}. For more information, see :ref:`intellij-logs`.
+
+   .. step:: Submit the ticket.
+
+      Click :guilabel:`Create` to submit the ticket.
+
+Vote on Submitted Tickets
```

Comment:
[question]

Should we reframe this section as "Next Steps" like in the [Info Type example](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/task/#next-steps)?

Reply:
I'm leaning towards leaving this as is. I considered the "Next Steps" option to match with the task page prototype, but I felt that: 
1. It'd be more clear/straightforward to have the "Vote on Submitted Tickets" section instead of a "Next Steps" header that isn't as descriptive with the body content in it. 
2. It'd be weird to have an H1 "Next Steps" immediately followed by an H2 "Vote on Submitted Tickets" when that's the only content really available for next steps.
3. There wouldn't be enough information on voting for tickets to write out in a separate reference page. 

LMK if you have other thoughts!

Reply:
Well stated! Thanks for sharing your rationale.